src/Markdownify/ConverterExtra.php

```diff
@@ -269,6 +269,9 @@ protected function handleTag_table()
         } else {
             // finally build the table in Markdown Extra syntax
             $separator = array();
+            if (!isset($this->table['aligns'])) {
+                $this->table['aligns'] = [];
```

Comment:
We support PHP5.3 for now.
Yes... I know... But I don't want to introduce a breaking change before a big rewrite of this library ;)